web/profiles/joinup/joinup.behat.inc

```diff
@@ -770,22 +770,13 @@ class JoinupSubContext extends DrupalSubContextBase implements DrupalSubContextI
    * @Then I( should) see the( following) tiles in the :region region:
    */
   public function assertTilesInBlock(TableNode $titles_table, $region) {
-    $session = $this->getSession();
-    $regionObj = $session->getPage()->find('region', $region);
-    if (!$regionObj) {
-      throw new \Exception(sprintf('No region "%s" found on the page %s.', $region, $session->getCurrentUrl()));
-    }
-
-    $results = $regionObj->findAll('css', '.listing__item--tile h2');
+    $regionObj = $this->getRegion($region);
+    $results = $regionObj->findAll('css', '.listing__item--tile .listing__title');
```

Comment:
You introduced the `getTiles()` method but never used it. I think it was meant for this part, right?